### ISSUE TITLE
Only count a student late when they are actually late

### ISSUE THREAD

**Issue by nwarmenh** [PSCS-Coding/Attendance_System#213]
Right now, in a student's reports, a student's late count will increment by one when the following happens: 
* Student notifies the school at (e.g.) 8:50 that they won't be arriving until 9:05
* Student arrives earlier than they expected at 8:58 and signs in as present

In a related case, when a student *doesn't* inform the school that they will be late, but also doesn't show up and sign in until 9:02 am, there is no "late" event in the database, and therefore they don't get counted as late.

In short, I think the "you have been late" line at the top of reports should be functionally synonymous with "you weren't present at 9am".

**Comment by badtzmaru** [comment]
closing this in favor of #214
